server/relay_entry_test.go

```diff
@@ -45,23 +45,23 @@ func TestParseRelaysURLs(t *testing.T) {
 			expectedURL:       "http://" + publicKey.String() + "@foo.com",
 		},
 		{
-			name:     "Relay URL with public key host and port",
+			name:     "Relay URL with public key host and Port",
```

Comment:
why?

Reply:
really not sure... maybe something weird as part of a save actions rule i've got setup in in goland i think... will get this revised 👍